### PR TITLE
Bug 1402483: Remove BVC logging spam

### PR DESCRIPTION
**Pull request by garvankeeley**
https://bugzilla.mozilla.org/show_bug.cgi?id=1402483

### CODE REVIEW

File: Client/Frontend/Browser/BrowserViewController.swift [hunk 1]
```diff
@@ -989,7 +957,7 @@ class BrowserViewController: UIViewController {
         profile.bookmarks.modelFactory >>== {
             $0.isBookmarked(url).uponQueue(DispatchQueue.main) { [weak tab] result in
                 guard let bookmarked = result.successValue else {
-                    log.error("Error getting bookmark status: \(result.failureValue ??? "nil").")
+                    print("Error getting bookmark status: \(result.failureValue ??? "nil").")
```
Comment (farhanpatel) [inline Client/Frontend/Browser/BrowserViewController.swift]: curious why you decided to switch it to print?

**Comment by farhanpatel** [comment]
There are also debug statements in TabManager, HistoryPanel, BookmarksPanel could we also remove those?

**Comment by farhanpatel** [comment]
tbh, and a lot of the debug statements in AppDelegate, we could probably get rid of those. At least the ones that always happen like `  log.debug("Making window key and visible…"), log.debug("Triggering log roll.")` 
We could keep the start/finish. but everything inbetween seems useless imo

File: Client/Frontend/Browser/BrowserViewController.swift (hunk 1 above)
Comment (garvankeeley) [reply Client/Frontend/Browser/BrowserViewController.swift]: I removed XCGLogger dependency for the class. XCGLogger is overkill except for sync and share debugging IMO.

**Comment by garvankeeley** [comment]
Agreed on above points, I'll take the others you mentioned out also.

File: Client/Application/AppDelegate.swift [hunk 2]
```diff
@@ -194,7 +179,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UIViewControllerRestorati
 
         setUpDeepLinks(application: application)
 
-        log.debug("Done with setting up the application.")
+        log.info("startApplication end")
```
Comment (garvankeeley) [inline Client/Application/AppDelegate.swift]: log.info is handy in one regard that it prints timings.

**Pull request merged**